omnigibson/prims/xform_prim.py

```diff
@@ -255,16 +261,7 @@ def get_2d_orientation(self):
         Get this prim's orientation on the XY plane of the world frame. This is obtained by
         projecting the forward vector onto the XY plane and then computing the angle.
         """
-        fwd = R.from_quat(self.get_orientation()).apply([1, 0, 0])
```

Comment:
nit: should we rename to `get_xy_orientation()` so that it's clear which 2 dimensions are being taken into account?